readme_renderer/clean.py

```diff
@@ -52,19 +50,43 @@
     "h4": ["align"],
     "h5": ["align"],
     "h6": ["align"],
-    "code": ["class"],
     "p": ["align"],
 }
+# Class is a specific attribute because not only do we want to allow it only
+# on certain tags, but we also want to control possible values.
+ALLOWED_CLASSES = {
+    "img": {"align-left", "align-right", "align-center"},
```

Comment:
If you look at https://github.com/pypa/warehouse/blob/master/warehouse/static/sass/blocks/_project-description.scss#L207-L209, you may find a discrepancy:

Warehouse CSS will align on 3 different conditions:
- `img.align-left` which is the one we target here
- `.figure.align-left` which according to https://github.com/pypa/warehouse/pull/4391#issuecomment-408669826 will be applied on a `div` element, but it's already broken as of today because `div` element doesn't allow `class` without our current bleaching rules, even before this PR. This could be fixed in a subsequent PR
- `object.align-left` but `object` tags are not even allowed!
So authorizing only those classes and only on `img` seems to not be a restriction compared to what we have today.